indico/web/flask/blueprints/event/management/abstracts.py

```diff
@@ -19,144 +19,144 @@
 
 
 # Setup
-event_mgmt.add_url_rule('/call-for-abstracts/setup/', 'confModifCFA', conferenceModif.RHConfModifCFA)
-event_mgmt.add_url_rule('/call-for-abstracts/setup/toggle', 'confModifCFA-changeStatus',
+event_mgmt.add_url_rule('/call-for-abstracts-old/setup/', 'confModifCFA', conferenceModif.RHConfModifCFA)
```

Comment:
no need to add `-old` here - the new CFA urls use `/abstracts/` instead of `/call-for-abstracts/`